src/cromero/BowlingGame.test.ts

```diff
@@ -0,0 +1,51 @@
+import { BowlingGame } from "./BowlingGame";
+
+describe(BowlingGame.name, function () {
+  it("should register pins by roll", function () {
+    const game = new BowlingGame();
+    game.roll(1);
+    game.roll(1);
+    game.roll(1);
+    game.roll(1);
+    expect(game.getScore()).toEqual(4);
+  });
+
+  it("should calculate score with strikes bonus", function () {
+    const game = new BowlingGame();
+    game.roll(10);
+    game.roll(5);
+    game.roll(3);
+    const score = game.getScore();
+    expect(score).toEqual(26);
+  });
+
+  it("should calculate score with spare bonus", function () {
+    const game = new BowlingGame();
+    game.roll(5);
+    game.roll(5);
+    game.roll(3);
+    game.roll(4);
+    const score = game.getScore();
+    expect(score).toEqual(20);
+  });
+
+  it("should get a extra rol in the 10th frame", function () {
+    const game = new BowlingGame();
+    game.roll(10);
+    for (let i = 0; i < 19; i++) {
+      game.roll(1);
+    }
+    const score = game.getScore();
+    expect(score).toEqual(31);
```

Comment:
I think that this is not a valid scenario because you do not have the extra roll in this game: 

![image](https://user-images.githubusercontent.com/1157864/177581831-fa15260e-31ff-4218-8bc6-fb36287cb52e.png)

You have an extra roll when you have a strike or spare in the last frame:

![image](https://user-images.githubusercontent.com/1157864/177582846-45ae52c2-5bdd-4864-ae24-04135274989b.png)